### ISSUE TITLE
Inference time for stable diffusion v1.4 checkpoint

### ISSUE THREAD

**Issue by Maitreyapatel** [soominkwon/resample#6]
Hello, 
Thanks for the great work!
I could quickly extend the script to work on regular LDM checkpoints. And the results are great!

However, it takes ~24 minutes for generation on A100 GPU. 
Is this expected? 

Best,

**Comment by efzero** [comment]
No, that should not be that long. Probably things go wrong on the optimization side

**Comment by Maitreyapatel** [comment]
Thanks for replying!

I get ~10 minutes by following the base script, which is at a lower resolution, so the above numbers make sense. 
I wonder if you have observed such behavior due to the type of GPU.  

Also, we are having issues with box inpainting, similar to #7 . We would appreciate it if you could release the corresponding script, too. 

I would love to get more details into this. We are using this work as a baseline and would like to report the actual numbers that we should get. Otherwise, it won't be a fair comparison.
